net-irc/znc/znc-9999.ebuild

```diff
@@ -2,163 +2,170 @@
-	dodoc NOTICE README.md
+	cmake-utils_src_install
+	dodoc NOTICE
 	if use daemon; then
-		newinitd "${FILESDIR}"/znc.initd-r1 znc
+		newinitd "${FILESDIR}"/znc.initd-r2 znc
 		newconfd "${FILESDIR}"/znc.confd-r1 znc
 	fi
+	DOC_CONTENTS=$(<"${FILESDIR}/README.gentoo") || die
+	DISABLE_AUTOFORMATTING=1
+	readme.gentoo_create_doc
 }
 
 pkg_postinst() {
-	if use !daemon; then
-		elog
-		elog "Run 'znc --makeconf' as the user you want to run ZNC as"
-		elog "to make a configuration file"
-		elog
-	else
-		elog
-		elog "An init-script was installed in /etc/init.d"
-		elog "A config file was installed in /etc/conf.d"
-		if [[ ! -d "${EROOT}${CONFDIR}" ]]; then
-			elog
-			elog "Run 'emerge --config znc' under portage"
-			elog "or 'cave config znc' under paludis to configure ZNC"
-			elog "as a system-wide daemon."
-			elog
-			elog "To generate a new SSL certificate, run:"
-			elog "  znc --system-wide-config-as znc --makepem -d ${CONFDIR}"
-			elog "as root"
-			elog
-			elog "If migrating from a user-based install"
-			elog "you can use your existing config files:"
-			elog "  mkdir ${CONFDIR}"
-			elog "  mv /home/\$USER/.znc/* ${CONFDIR}"
-			elog "  rm -rf /home/\$USER/.znc"
-			elog "  chown -R znc:znc ${CONFDIR}"
-			elog
-			elog "If you already have znc set up and want take advantage of the"
-			elog "init script but skip of all the above, you can also edit"
-			elog "  /etc/conf.d/znc"
-			elog "and adjust the variables to your current znc user and config"
-			elog "location."
-			if [[ -d "${EROOT}"/etc/znc ]]; then
-				elog
-				ewarn "/etc/znc exists on your system."
-				ewarn "Due to the nature of the contents of that folder,"
-				ewarn "we have changed the default configuration to use"
-				ewarn "	/var/lib/znc"
-				ewarn "please move /etc/znc to /var/lib/znc"
-				ewarn "or adjust /etc/conf.d/znc"
-			fi
-		else
-			elog "Existing config detected in ${CONFDIR}"
-			elog "You're good to go :)"
-		fi
-		elog
+	readme.gentoo_print_elog
+	if [[ -d "${EROOT%/}"/etc/znc ]]; then
+		ewarn "/etc/znc exists on your system."
+		ewarn "Due to the nature of the contents of that folder,"
+		ewarn "we have changed the default configuration to use"
+		ewarn "	/var/lib/znc"
+		ewarn "please move /etc/znc to /var/lib/znc"
+		ewarn "or adjust /etc/conf.d/znc"
 	fi
 }
 
 pkg_config() {
-	if use daemon && ! [[ -d "${EROOT}${CONFDIR}" ]]; then
-		einfo "Press ENTER to interactively create a new configuration file for znc."
-		einfo "To abort, press Control-C"
-		read
-		mkdir -p "${EROOT}${CONFDIR}" || die
-		chown -R ${PN}:${PN} "${EROOT}${CONFDIR}" ||
-			die "Setting permissions failed"
-		"${EROOT}"/usr/bin/znc --system-wide-config-as znc -c -r -d "${EROOT}${CONFDIR}" ||
-			die "Config failed"
-		echo
-		einfo "To start znc, run '/etc/init.d/znc start'"
-		einfo "or add znc to a runlevel:"
-		einfo "  rc-update add znc default"
-	else
-		if use daemon; then
-			ewarn "${CONFDIR} already exists, aborting to avoid damaging"
+	if use daemon; then
```

Comment:
Depending on your answer before we maybe don't need the daemon USE flag anymore.